README.md

```diff
@@ -139,11 +143,31 @@ ensure
 end
 ```
 
-## Customization
+## Validations
 
-If you need custom relationship handling (for instance, if you have a relationship named `scissors` that is plural, but it actually is a single entity), you can use Handlers to define appropriate behaviour.
+JsonApi::Parameters is validating your payloads **ONLY** when an error occurs. **This means that unless there was an exception, your payload will not be validated.** 
 
-Read more at [Relationship Handlers](https://github.com/visualitypl/jsonapi_parameters/wiki/Relationship-handlers).
+Reason for that is we prefer to avoid any performance overheads, and in most cases the validation errors will only be useful in the development environments, and mostly in the early parts of the implementation process. Our decision was to leave the validation to happen only in case JsonApi::Parameters failed to accomplish its task. 
+
+The validation happens with the use of jsonapi.org's JSON schema draft 6, available [here](https://jsonapi.org/faq/#is-there-a-json-schema-describing-json-api), and a gem called [JSONSchemer](https://github.com/davishmcclurg/json_schemer).
+
+If you would prefer to suppress validation errors, you can do so by declaring it globally in your application: 
+
+```ruby
+# config/initializers/jsonapi_parameters.rb
+
+JsonApi::Parameters.suppress_validation_errors = true
```

Comment:
`suppress_validation_errors` sounds like it supresses ActiveRecord validations. Name should suggest that this is only schema validation

Reply:
👍

Reply:
and same for `enforce_prevalidation`

Reply:
Sounds good, I'll change it to `suppress_schema_validation_errors` then :+1:

Reply:
consider also changing `enforce_prevalidation` to `enforce_schema_prevalidation`

Reply:
@ipepe yes, I agree and will change both of these :+1: